dts/bindings/dac/ti,dacx0508-base.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2020 M2I Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+include: [dac-controller.yaml, spi-device.yaml]
+
+properties:
+    voltage-reference:
+      type: int
+      required: true
+      description: DAC voltage reference select
+
+    channel0-gain:
+      type: int
+      required: true
+      description: Channel 0 gain select
```

Comment:
Perhaps add a header under `include/dt-bindings/dac/` for definitions on what goes into this property?

Also, it would be nice with a sample - or at least a compile-time test - of this driver. Is it available on a shield, or ...?